src/Microsoft.DotNet.Build.Tasks.Feed/src/PublishSymbolsHelper.cs

```diff
@@ -0,0 +1,158 @@
+                int expirationInDays,
+                bool convertPortablePdbsToWindowsPdbs,
+                bool publishSpecialClrFiles,
+                HashSet<int> pdbConversionTreatAsWarning,
+                bool dryRun,
+                bool timer,
+                bool verboseLogging)
+            {
+                var tracer = new Tracer(log, verboseLogging);
+
+                PublishOperation publishOperation = new PublishOperation(tracer)
+                {
+                    SymbolServerPath = symbolServerPath,
+                    PersonalAccessToken = personalAccessToken,
+                    PdbConversionTreatAsWarning = pdbConversionTreatAsWarning,
+                    PublishSpecialClrFiles = publishSpecialClrFiles,
+                    Timer = timer,
+                };
+
+                using (publishOperation)
+                {
+                    if (expirationInDays != 0)
+                    {
+                        publishOperation.ExpirationInDays = (uint) expirationInDays;
+                    }
+
+                    IEnumerable<PublishFileInfo> fileInfos = new PublishFileInfo[0];
+                    if (inputFiles != null)
+                    {
+                        fileInfos = fileInfos.Concat(
+                            publishOperation.GetPublishFileInfo(inputFiles, convertPortablePdbsToWindowsPdbs));
+                    }
+
+                    if (inputPackages != null)
+                    {
+                        fileInfos = fileInfos.Concat(
+                            publishOperation.GetPublishFileInfoFromPackages(inputPackages,
+                                convertPortablePdbsToWindowsPdbs));
+                    }
+
+                    if (packageExcludeFiles != null)
+                    {
+                        publishOperation.PackageExcludeFiles = packageExcludeFiles;
+                    }
+
+                    if (dryRun)
+                    {
+                        publishOperation.StartTimer();
+                        try
+                        {
+                            foreach (PublishFileInfo fileInfo in fileInfos)
+                            {
+                                fileInfo.Dispose();
+                            }
+                        }
+                        catch (Exception ex)
+                        {
+                            tracer.Error("Dry Run FAILED: {0}", ex.Message);
+                            tracer.Information(ex.ToString());
+                        }
+
+                        publishOperation.StopTimer();
+                    }
+                    else
+                    {
+                        publishOperation.PublishFiles(fileInfos).GetAwaiter().GetResult();
+                    }
+                }
+            }
+        }
+
+    internal class Tracer : Microsoft.SymbolStore.ITracer
+    {
+        readonly TaskLoggingHelper m_log;
+        readonly bool m_verbose;
```

Comment:
```suggestion
        readonly TaskLoggingHelper _log;
        readonly bool _verbose;
```

I believe the convention is to not use m_ to name these